docs/clustering/clustering_evaluation.md

```diff
@@ -83,11 +83,13 @@ X, y = datasets.make_blobs(
     shuffle=True,
     random_state=1,
 )
-
+cluster_labels = []
 kmeans = KMeans(n_clusters=4, random_state=1, n_init=5)
-cluster_labels = kmeans.fit_predict(X)
+cluster_labels.append(kmeans.fit_predict(X))
+kmeans = KMeans(n_clusters=5, random_state=1, n_init=5)
+cluster_labels.append(kmeans.fit_predict(X))
```

Comment:
let's move the `_from_results` methods (elbow and silhouette) out of this and put them as docstring examples (if we don't have them). then, let's link from here to the docstring so people know they exist. something like:

~~~md

```{tip}
If you want to train the models yourself, you can use `from_results` to plot.
```
~~~

there's a way to link directly to a class definition without a hardcoded link, I think `:class:`, but I'm unsure if that will work with the markdown format,let's try and see. if it doesn't work, we can use raw rst format: https://jupyterbook.org/en/stable/file-types/restructuredtext.html

Reply:
I think :class: directive works with classes but this one's a function.
I was able to link using heading labels like so
```
.. _silhouette-analysis-from-results-label:

silhouette_analysis_from_results
--------------------------------
.. autofunction:: sklearn_evaluation.plot.silhouette_analysis_from_results
```

and then using this label to create a link in the user guide. But the `tip` or `note` directives are not showing up. Still trying to find the issue

Reply:
Edit: it was an indentation issue. Will push the change